components/CodeBlockSection.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { CopyBlock, nord } from "react-code-blocks";
```

Comment:
can you remove `nord` as it is not used.
```suggestion
import { CopyBlock } from "react-code-blocks";
```